scout/utils/scout_requests.py

```diff
@@ -4,7 +4,7 @@
 import urllib.request
 import zlib
 from urllib.error import HTTPError
-from xml.etree import ElementTree
+from defusedxml import ElementTree
```

Comment:
Aha, is this new stdlib?

Reply:
Hadn't seen it - it was a recommendation from CodeFactor!